Tasks/ContainerStructureTestV1/task.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "39bc2c9b-55b7-4835-89cd-6cc699ef7220",
+    "name": "ContainerStructureTest",
+    "friendlyName": "Container Structure Test",
+    "description": "Runs Container structure tests using Google container structure test framework",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-test-results",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613742)",
+    "category": "Test",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.0.0",
+    "inputs": [
+        {
+            "name": "testFile",
+            "type": "multiline",
+            "label": "Test Files",
+            "required": true,
+            "helpMarkDown": "Test files path, that contains container structure tests. Either .yaml or .json files"
+        },
+        {
+            "name": "image",
```

Comment:
Check Docker task for inputs - Use 3 inputs - dockerRegistryServiceConnection (in Docker task it's called containerRegistry; But call the input dockerRegistryServiceConnection as we got some feedback), repository and tags.

Reply:
I have used it as place holders. I am not using it right now. Will sync up with you and Vikranth on how to use some these.